options/locale/locale_en-US.ini

```
@@ -1478,9 +1486,16 @@ pulls.outdated_with_base_branch = This branch is out-of-date with the base branc
 pulls.closed_at = `closed this pull request <a id="%[1]s" href="#%[1]s">%[2]s</a>`
 pulls.reopened_at = `reopened this pull request <a id="%[1]s" href="#%[1]s">%[2]s</a>`
 pulls.merge_instruction_hint = `You can also view <a class="show-instruction">command line instructions</a>.`
-
 pulls.merge_instruction_step1_desc = From your project repository, check out a new branch and test the changes.
 pulls.merge_instruction_step2_desc = Merge the changes and update on Gitea.
+pulls.merge_on_status_success_success = The pull request was successfully scheduled to merge when all checks succeed.
```

You can unsuccessfully schedule to merge when all checks succeeded?
I think this should be omitted and only mentioned in case the scheduling failed.
```suggestion
pulls.merge_on_status_success_success = The pull request was scheduled to merge when all checks succeed.
```